tools/mpremote/mpremote/main.py

```diff
@@ -439,6 +439,11 @@ def main():
         while args:
             do_command_expansion(args)
             cmd = args.pop(0)
+            
+            if cmd == "help" or cmd == "--help":
```

Comment:
Would it be simpler to add a shortcut to `_BUILTIN_COMMAND_EXPANSIONS` that just maps `--help` to `help`?

Reply:
Oh, I see what you mean. I hadn't seen `_BUILTIN_COMMAND_EXPANSIONS ` / how it operates - yes that looks like it would be simpler!

Reply:
Yep that change worked, has been pushed here instead thanks.